src/plugins/ai_assistant_management/selection/public/routes/components/ai_assistant_selection_page.tsx

```diff
@@ -24,10 +24,23 @@ import { useAppContext } from '../../app_context';
 
 export function AiAssistantSelectionPage() {
   const { capabilities, setBreadcrumbs, navigateToApp } = useAppContext();
-
-  const observabilityAIAssistantEnabled = capabilities.observabilityAIAssistant.show;
+  const observabilityAIAssistantPluginEnabled = capabilities.observabilityAIAssistant !== undefined;
+  const observabilityAIAssistantEnabled = capabilities.observabilityAIAssistant?.show;
   const securityAIAssistantEnabled = capabilities.securitySolutionAssistant?.['ai-assistant'];
 
+  const observabilityAIAssistantPluginDisabledMsg = i18n.translate(
+    'aiAssistantManagementSelection.aiAssistantSelectionPage.observabilityAi.thisFeatureIsDisabledCallOutLabel',
+    {
+      defaultMessage: 'This feature is disabled.',
+    }
+  );
+
+  const observabilityFeatureDisabledInSpaceMsg = i18n.translate(
+    'aiAssistantManagementSelection.aiAssistantSelectionPage.observabilityAi.thisFeatureIsDisabledInSpaceCallOutLabel',
+    {
+      defaultMessage: 'This feature is disabled. It can be enabled from Spaces > Features',
```

Comment:
Should we label this is as "Stack Management > Spaces > Features"? This is the title of the breadcrumb:

![CleanShot 2024-07-15 at 11 52 13@2x](https://github.com/user-attachments/assets/f425250c-fd20-40c2-b78e-5f68cc002ab3)

Reply:
@dgieselaar It was decided to keep the message simple instead of distinguishing between the states so it now only says "This feature is disabled" with the existing label.